packages/sawja.1.5.1/opam

```diff
@@ -0,0 +1,13 @@
+opam-version: "1.1.1"
+name: "sawja"
+maintainer: "sawja@inria.fr"
+version: "1.5.1"
+build: [
+  ["./configure.sh"]
+  [make]
+  [make "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "sawja"]
+]
+depends: [ "ocamlfind" "javalib" {= "2.3.1"} ]
```

Comment:
`ocamlfind` can be marked as a `{build}` dependency so that updating `ocamlfind` doesn't force a rebuild of `sawja`. Also, it's generally preferable to make the version constraints on dependencies as broad as reasonable for compatibility (e.g. `{> "2.3.0"}` perhaps).